libretro-common/glsm/glsm.c

```diff
@@ -2500,7 +2500,11 @@ void rglProgramBinary(GLuint program,
 
 void rglTexImage2DMultisample( 	GLenum target,
   	GLsizei samples,
+#if defined(OSX)
+  	GLint internalformat,
```

Comment:
I need to double check them, I doubt it differs for IOS so its probably wrong type to begin with